code/modules/paperwork/photocopier.dm

```diff
@@ -120,7 +120,7 @@
 		else if(doccopy)
 			for(var/i = 0, i < copies, i++)
 				if(toner > 5 && !busy && doccopy)
-					new /obj/item/documents/photocopy(src, doccopy)
+					new /obj/item/documents/photocopy(loc)

```

Comment:
No, because if you have _checked the code _before removing "unused" variables, this conversation wouldn't even happen.